### ISSUE TITLE
hide closed cases

### ISSUE THREAD

**Issue by arroyoDev** [CrisisCleanup/crisiscleanup-3-web#416]
I think it would be useful to have a "hide closed cases" checkbox.   keeps important stuff up front.   

- [ ] add a "hide closed cases" checkbox.  

<img width="1353" alt="image" src="https://user-images.githubusercontent.com/699333/77493740-d5483500-6e11-11ea-9e5f-ed4767f36efc.png">

**Comment by aarontitus** [comment]
Agree. As long as we are adding filters, we ought to consider whether there are any others that are essential.
